src/backend/web/graphql/queries.js

```diff
@@ -0,0 +1,99 @@
+const { gql } = require('apollo-server-express');
+
+const maxPostsPerPage = 30;
+
+module.exports.typeDefs = gql`
+  # 'Feed' matches our Feed type used with redis
+  type Feed {
+    name: String
+    url: String
+  }
+
+  # 'Post' matches our Post type used with redis
+  type Post {
+    author: String
+    title: String
+    html: String
+    text: String
+    published: String
+    updated: String
+    url: String
+    site: String
+    guid: String
+  }
+
+  # Queries to fetch data from redis
+  type Query {
+    getFeed(guid: ID!): Feed
+    getFeeds: [ID]
+    getFeedsCount: Float
+    getPost(guid: ID!): Post
+    getPosts(page: Int, perPage: Int): [Post]
+    getPostsCount: Float
+  }
+`;
+
+module.exports.resolvers = {
+  Query: {
+    /**
+     * @description Takes a guid and returns a Feed object
+     * @param guid
+     * @return Feed object for the passed guid
+     */
+    getFeed: (parent, { guid }, { storage }) => {
+      return storage.getFeed(guid);
+    },
+
+    /**
+     * @description Fetches all the Feed objects in our database
+     * @return Array Feed objects
+     */
+    getFeeds: (parent, arg, { storage }) => {
+      return storage.getFeeds('feeds');
+    },
+
+    /**
+     * @return Number of Feed objects in our database
+     */
+    getFeedsCount: (parent, arg, { storage }) => {
+      return storage.getFeedsCount();
+    },
+
+    /**
+     * @description Takes a guid and returns a Post object
+     * @param guid
+     * @return Post object for the passed guid
+     */
+    getPost: (parent, { guid }, { storage }) => {
+      return storage.getPost(guid);
```

Comment:
We can use `Post.byGuid()` here instead to get proper types: https://github.com/Seneca-CDOT/telescope/blob/master/src/backend/post.js#L125